Clean database naming ambiguity

albertoalmagro opened the pull request:
### Summary

When I was reading this file I found out that this file referred to the database in three different ways: database, DB and db. The last one caused confusion with the db namespace and the db folder. This patch removes this ambiguity by using the whole word 'database' everywhere

rafaelfranca commented on railties/lib/rails/tasks/engine.rake:
```diff
@@ -62,7 +62,7 @@ namespace :db do
   app_task "seed"
 
-  desc "Create the database, load the schema, and initialize with the seed data (use db:reset to also drop the db first)"
+  desc "Create the database, load the schema, and initialize with the seed data (use db:reset to also drop the DB first)"
```
let's change to database.

albertoalmagro replied:
@rafaelfranca thanks! you always give the things a better twist ❤️ 

I have changed it as suggested and also updated the PR title and description.

Commit:
Remove DB naming ambiguity

In some places this file referred to the database in three different ways: database, DB and db. The last one caused confusion with the db namespace and the db folder. This commit removes this ambiguity by using the whole word 'database' everywhere

georgeclaghorn commented:
r? @rafaelfranca

albertoalmagro commented:
Hi @rafaelfranca , changes are done, could you please r? Thanks!
